src/TabBarItem.tsx

```diff
@@ -45,7 +48,7 @@ type Props<T extends Route> = {
 const DEFAULT_ACTIVE_COLOR = 'rgba(255, 255, 255, 1)';
 const DEFAULT_INACTIVE_COLOR = 'rgba(255, 255, 255, 0.7)';
 
-export default class TabBarItem<T extends Route> extends React.Component<
+export default class TabBarItem<T extends Route> extends React.PureComponent<
```

Comment:
Is changing to `PureComponent` needed here?

Reply:
reverted